django/core/management/utils.py

```diff
@@ -79,7 +79,7 @@ def get_random_secret_key():
     Return a 50 character random string usable as a SECRET_KEY setting value.
     """
     chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
-    return get_random_string(50, chars)
+    return 'dj::insecure' + get_random_string(38, chars)
```

Comment:
Maybe ``return 'dj::insecure-' + get_random_string(38, chars)`` to improve the readability of the value a little?

Also, I think we should probably keep the 50 random characters so we don't lose randomness in comparison to the current behavior.

Reply:
Oh, it's ok. I've though about - sign but decided to do exactly as in ticket. I agree about keep 50 signs, so I will edit code and the tests checking default SECRET_KEY's length very soon. Thanks a lot for your review!